src/SFA.DAS.AssessorService.ApplyTypes/ApplicationStatus.cs

```diff
@@ -9,5 +9,6 @@ public static class ApplicationStatus
         public const string Resubmitted = "Resubmitted";
         public const string Declined = "Declined";
         public const string Approved = "Approved";
+        public const string GatewayAssessed = "GatewayAssessed";
```

Comment:
Mentioned elsewhere.. we need to figure out if it goes here. I'm implementing a new Column called GatewayReviewStatus